docs/topics/api/addons.rst

```diff
@@ -159,6 +159,7 @@ This endpoint allows you to fetch a specific add-on by id, slug or guid.
     :>json object categories: Object holding the categories the add-on belongs to.
     :>json array categories[app_name]: Array holding the :ref:`category slugs <category-list>` the add-on belongs to for a given :ref:`add-on application <addon-detail-application>`. (Combine with the add-on ``type`` to determine the name of the category).
     :>json string|null contributions_url: URL to the (external) webpage where the addon's authors collect monetary contributions, if set.
+    :>json string created: The date the add-on got created.
```

Comment:
"add-on _was_ created" is generally considered more formal (and matches how `last_updated` is phrased)